email-confirmation/functions/index.js

```diff
@@ -45,6 +45,8 @@ exports.sendEmailConfirmation = functions.database.ref('/users/{uid}').onWrite(e
     mailOptions.text = 'Thanks you for subscribing to our newsletter. You will receive our next weekly newsletter.';
     return mailTransport.sendMail(mailOptions).then(() => {
       console.log('New subscription confirmation email sent to:', val.email);
+    }).catch(error => {
+      console.log(error);  
```

Comment:
just do `console.error('There was an error while sending the email:', error)` here